proposed/phpdoc.md

```diff
@@ -461,7 +461,7 @@ A Type has the following [ABNF][RFC5234] definition:
     class-name       = ["\"] label *("\" label)
     label            = (ALPHA / %x7F-FF) *(ALPHA / DIGIT / %x7F-FF)
     keyword          = "array" / "bool" / "callable" / "false" / "float" / "int" / "iterable" / "mixed" / "null" / "object" /
-    keyword          = "resource" / "self" / "static" / "string" / "true" / "void" / "$this"
+    keyword          = "resource" / "self" / "static" / "string" / "true" / "void" / "$this" / "no-return"
```

Comment:
```suggestion
    keyword          = "resource" / "self" / "static" / "string" / "true" / "void" / "$this" / "noreturn"
```

Rename to match the [noreturn_type PHP RFC](https://wiki.php.net/rfc/noreturn_type)

Reply:
```suggestion
    keyword          = "resource" / "self" / "static" / "string" / "true" / "void" / "$this" / "never"
```